core/templates/domain/exploration/SolutionObjectFactorySpec.ts

```diff
@@ -26,15 +26,15 @@ import {
 import { ConvertToPlainTextPipe } from
```

Comment:
I think this file should also be renamed.

Reply:
ok, I will do it.